src/components/SUE/report/SueReportDescription.tsx

```diff
@@ -11,6 +11,8 @@ import { Input } from '../../form';
 
 const { IMAGE_SELECTOR_TYPES, IMAGE_SELECTOR_ERROR_TYPES, INPUT_KEYS } = consts;
 
+const MemoizedImageSelector = React.memo((props) => <ImageSelector {...props} />);
```

Comment:
please import memo and use it directly instead of from React like in the other file src/screens/SUE/SueReportScreen.tsx